test_scripts/API/AutoCompleteList/003_ATF_SetGlobalProperties_N_transfer_NotAllowed_request.lua

```diff
@@ -0,0 +1,185 @@
+-----------------------------------------------------------------------------------------------------------------------
+-- Requirement summary:
+-- [SetGlobalProperties] Conditions for SDL SDL respond <success = false, resultCode = "DISALLOWED"> to mobile app
+--
+-- Description:
+-- SDL doesn’t transfer not allowed "SetGlobalProperties"_request with valid ""autoCompleteList" param to HMI
+-- respond with <resultCode_received_from _HMI> to mobile app
+--
+-- Performed steps:
+-- 1. Register Application.
+-- 2. Mobile send RPC SetGlobalProperties with <autoCompleteList>.
+-- 3. SDL respond <success = true, resultCode = "DISALLOWED"> to mobile app
```

Comment:
@Aleksandriys, looks like you missed "Expected result"(mandatory in template) in header.
I guess, at least p3." SDL respond <success = true, resultCode = "DISALLOWED"> to mobile app" Should be in expected result.